src/frontend/src/styles/main.css

```diff
@@ -1990,7 +1990,7 @@ a:hover,
 }
 
 .c-chasm__arrow {
-  margin: auto;
+  margin-left: calc(1.25 * var(--rs-card-bezel));
```

Comment:
side note:

@meodai `1.25 * var(--rs-card-bezel)` comes few times in the CSS file, four exactly, should a variable be extracted?